src/cider/nrepl/middleware/debug.clj

```diff
@@ -117,12 +117,19 @@
   "Bound by the `breakpoint` macro to the local &env."
   {})
 
+(defn pr-short
+  "Like `pr-str` but limited in length and depth."
+  [x]
+  (binding [*print-length* 6
```

Comment:
Pushed an updated that lets us specify these values when the debugger is initialized. I'll make the cider.el counterpart soon.